### DISCUSSION TITLE
Why am I banned from the Telegram group?

### DISCUSSION THREAD

**Discussion by L1keava** [ChefKissInc/NootedRed#223] [Q&A] [answered]
I used to be a member of a Telegram group, but I can no longer use the invitation link to join the group. If I am banned from joining the group, can you lift the ban for me? (@Likeavar) I don't understand what caused this, and if it was my inappropriate words and deeds that caused this, I'm very sorry for it. Thx

**Comment by VisualEhrmanntraut** [comment, marked as answer]
The Telegram supergroup has its own set of rules and preflight checks (than the rules of other communities and platforms). This is in order to maintain quality of the content and communication therein. You can still continue using our software and posting in the discussions, creating issues, etc. as long as you're not causing any issues (spam, etc), in which case you'd be blocked from the org.
If you'd like to follow updates, you can subscribe/watch the repos and receive notifications that way. In the upcoming v2 website, there will also be a blog you can subscribe to using RSS.
